hnn_core/cell.py

```diff
@@ -97,9 +97,9 @@ class _Cell(ABC):
         Contains recording of somatic currents indexed
         by synapse type. (keys are soma_gabaa, soma_gabab etc.)
         Must be enabled by running simulate_dipole(net, record_isoma=True)
-    tonic_feeds : list of h.IClamp
+    tonic_biases : list of h.IClamp
         The current clamps inserted at each section of the cell
-        for tonic inputs.
+        for tonic biasing inputs.
```

Comment:
Are we planning on implementing multiple tonic biases in the near future? It seems to me that at least at the cellular level, this should be a singular value, `tonic_bias`.

Reply:
Good point. As it stands, the `h.IClamp` is only ever applied to the soma, so there really is no point in the plural form. And it's doubtful whether clamps along the dendrite are ever going to be allowed into HNN. So I'm +1 for renaming. In addition:

- the attribute `tonic_biases` (list) is never accessed -> remove? Or is it somehow implicitly important for Neuron? Does `stim = h.IClamp(self.soma(loc)` apply the clamp?
- I propose renaming `cell.create_tonic_bias` to `cell.apply_tonic_bias`?

Reply:
>     * the attribute `tonic_biases` (list) is never accessed -> remove? Or is it somehow implicitly important for Neuron? Does `stim = h.IClamp(self.soma(loc)` apply the clamp?

I can't help but wonder if storing a reference to `stim = h.IClamp(self.soma(loc)` (as an object attribute) is important for maintaining access to the hoc object. Maybe we should test it somehow. Is it possible to access instantiated `h.IClamp()` properties at some other level of the network?

>     * I propose renaming `cell.create_tonic_bias` to `cell.apply_tonic_bias`?

I agree. 'apply' is more descriptive in this case.